### PR TITLE
Lexiswest PR

### PR DESCRIPTION
**Pull request by flooie**
Major update to the reporters-db

400+ commits
5k new lines to the reporters-db

### CODE REVIEW

File: reporters_db/data/reporters.json [hunk 1]
```diff
@@ -1310,6 +2058,10 @@
                 "Cal. 4th": {
                     "end": null,
                     "start": "1991-01-01T00:00:00"
+                },
+                "Cal. 5th": {
+                    "end": null,
+                    "start": "1750-01-01T00:00:00"
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: This can at least start in 1991.

File: reporters_db/data/reporters.json [hunk 2]
```diff
@@ -1342,6 +2116,10 @@
                 "Cal. App. 4th": {
                     "end": null,
                     "start": "1991-01-01T00:00:00"
+                },
+                "Cal. App. 5th": {
+                    "end": null,
+                    "start": "1750-01-01T00:00:00"
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: 1991 again here.

File: reporters_db/data/reporters.json [hunk 3]
```diff
@@ -1585,18 +2509,79 @@
...
+                    "end": "1925-12-31T00:00:00",
+                    "start": "1868-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:c7:il.nd;district.court"
+            ],
+            "name": "Chicago Legal News (1868-1925) (Illinois)",
+            "variations": {
+                "placeholder_0": "Chi. Leg. News"
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: This is...odd?

File: reporters_db/data/reporters.json [hunk 4]
```diff
@@ -69,18 +69,34 @@
...
                 "Ap.2d.": "A.D.",
-                "App.Div.": "A.D.",
                 "App. Div.": "A.D.",
-                "App.Div.2d.": "A.D.",
                 "App. Div. 2d.": "A.D.",
-                "App.Div.3d.": "A.D.",
                 "App. Div. 3d.": "A.D.",
+                "App.Div.": "A.D.",
                 "App.Div.(N.Y.)": "A.D.",
+                "App.Div.2d.": "A.D.",
```
[the remark this answers is not in the document]
Comment (flooie) [reply reporters_db/data/reporters.json]: yes ooops

File: reporters_db/data/reporters.json (hunk 1 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: done

File: reporters_db/data/reporters.json (hunk 2 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: done

File: reporters_db/data/reporters.json (hunk 3 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: yeah - removed-  this is an artifact from my adding script - no others existed.

**Comment by mlissner** [comment]
Looks good, but tests are failing. You might want to set them to auto-run in your editor because they're really fast. Can you get that fixed, then we're in business. Looks like something to do with `Or.` being a variation of itself. (I love the tests in this repo.)

**Pull request merged**